aliases/available/docker.aliases.bash

```diff
@@ -0,0 +1,16 @@
+cite 'about-alias'
+about-alias 'docker abbreviations'
+
+alias dklc='docker ps -l'  # List last Docker container
+alias dklcid='docker ps -l -q'  # List last Docker container ID
+alias dklcip="docker inspect `dklcid` | grep IPAddress | cut -d '\"' -f 4"  # Get IP of last Docker container
+alias dkps='docker ps'  # List running Docker containers
+alias dkpsa='docker ps -a'  # List all Docker containers
+alias dki='docker images'  # List Docker images
+alias dkrmac='docker rm $(docker ps -a -q)'  # Delete all Docker containers
+alias dkrmlc='docker-remove-most-recent-container'  # Delete most recent (i.e., last) Docker container
```

Comment:
One issue I can see with this (and I'm not blaming you for this): Some of the functions use a dash as a word separator, while others use an underscore.

I know we're not consistent with this in Bash-it (some plugins use the dash, others the underscore), but I think we should try to keep it as consistent as possible, at least within the plugin. Mixing syntax within the same plugin is not a good idea.

@tswicegood What's your view on this?